src/status_im/common/signals/events.cljs

```diff
@@ -84,7 +84,9 @@
       (local-notifications/process cofx (transforms/js->clj event-js))
 
       "community.found"
-      (link-preview/cache-community-preview-data (transforms/js->clj event-js))
+      (let [community (transforms/js->clj event-js)]
+        (link-preview/cache-community-preview-data community)
+        {:fx [[:dispatch [:discover-community/maybe-found-unknown-contract-community community]]]})
```

Comment:
sorry for the confusion, here you have to dispatch `cache-community-preview-data`  as well as it was in the first iteration of your PR.